src/backend/base/langflow/services/tracing/langwatch.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, trace_name: str, trace_type: str, project_name: str, trace_id
             self.trace = self._client.trace(
                 trace_id=str(self.trace_id),
             )
-            self.spans: dict[str, "ContextSpan"] = {}
+            self.spans: dict[str, ContextSpan] = {}
```

Comment:
This might not work because it is not imported if not type checking.

Reply:
Oh, right, I forgot that it needs `from __future__ import annotations` to be that way.